bin/settings/settings.env

```diff
@@ -70,7 +65,7 @@ UAADB_PASSWORD=uaaadmin_password
 UAA_ADMIN_CLIENT_SECRET=admin_secret
 UAA_CC_CLIENT_SECRET=cc_client_secret
 UAA_CLIENTS_CC_ROUTING_SECRET=cc_routing_secret
-UAA_CLIENTS_CC_SERVICE_SECRET=cc_service_secret
+#UAA_CLIENTS_CC_SERVICE_SECRET=cc_service_secret
```

Comment:
Why is this being removed?

Reply:
Because the mysql broker went away

Reply:
Then this still needs to live in the _mysql broker went away_ PR :)